### PR TITLE
OKTA-296678: Updates System Log API docs to include mention about time references

### PR DESCRIPTION
**Pull request by bobtiernay-okta**
## Description:
- **What's changed?** Updates System Log API docs to include mention about time references
- **Is this PR related to a Monolith release?** No

image
image
image

### Resolves:

* [OKTA-296678](https://oktainc.atlassian.net/browse/OKTA-296678)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/system-log/index.md [hunk 1]
```diff
@@ -624,6 +625,7 @@ For a request to be a _bounded_ request it must meet the following request param
   - `until` must be specified.
 
 Bounded requests to the `/api/v1/logs` API have the following semantics:
+  - They are time-filtered by their `published` field (unlike [Polling Requests](#polling-requests)).
```
Comment (ryanoblak-okta) [inline packages/@okta/vuepress-site/docs/reference/api/system-log/index.md:628]: Nit: consider "event `published`" to differentiate it from possible confusion with a request query parameter.

File: packages/@okta/vuepress-site/docs/reference/api/system-log/index.md [hunk 2]
```diff
@@ -609,6 +609,7 @@ For a request to be a _polling_ request it must meet the following request param
 
 Polling requests to the `/api/v1/logs` API have the following semantics:
   - They return every event that occurs in your organization.
+  - They are time-filtered by their internal "persistence time" to avoid skipping records due to system delays (unlike [Bounded Requests](#bounded-requests)).
```
Comment (ryanoblak-okta) [inline packages/@okta/vuepress-site/docs/reference/api/system-log/index.md:612]: Nit: consider replacing "They" with "Events" for clarity/differentiation from requests.

File: packages/@okta/vuepress-site/docs/reference/api/system-log/index.md [hunk 3]
```diff
@@ -609,6 +609,7 @@ For a request to be a _polling_ request it must meet the following request param
 
 Polling requests to the `/api/v1/logs` API have the following semantics:
   - They return every event that occurs in your organization.
+  - The returned events are time-filtered by their internal "persistence time" to avoid skipping records due to system delays (unlike [Bounded Requests](#bounded-requests)).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/system-log/index.md:612]: time-filtered -> time filtered

this doesn't need to be hyphenated :)

File: packages/@okta/vuepress-site/docs/reference/api/system-log/index.md [hunk 4]
```diff
@@ -624,6 +625,7 @@ For a request to be a _bounded_ request it must meet the following request param
   - `until` must be specified.
 
 Bounded requests to the `/api/v1/logs` API have the following semantics:
+  - The returned events are time-filtered by their associated `published` field (unlike [Polling Requests](#polling-requests)).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/system-log/index.md:628]: time-filtered -> time filtered

this doesn't need to be hyphenated :)

**Pull request merged**